daemon/algod/api/algod.oas2.json

```diff
@@ -1255,13 +1256,12 @@
         "operationId": "SimulateTransaction",
         "parameters": [
           {
-            "description": "The byte encoded transaction to simulate",
-            "name": "rawtxn",
+            "description": "The transactions to simulate, along with any other inputs.",
```

Comment:
nit:
```suggestion
            "description": "The transaction groups to simulate, along with any other inputs.",
```

Reply:
Since we don't yet support multiple groups, my preference is to keep it ambiguous with "transactions"

Reply:
okok, I was trying to see where we will fall into similar problem(s) down the road, so I am spamming some things here as a reference:
- we might want to change `goal clerk simulate`'s `-t` to be `StringArray*` or `StringSlice*` to accept multiple txn file for multi-group simulation.
- also update model description for transactions to transaction group(s).

If we are good about what we foresee, let's resolve it for now